src/components/TimeSeriesCard/TimeSeriesCard.story.jsx

```diff
@@ -695,6 +672,46 @@ storiesOf('Watson IoT/TimeSeriesCard', module)
       </div>
     );
   })
+  .add('domainRange', () => {
```

Comment:
What was this story added for? Is this intentional?

Reply:
See above comment